Alloy/template/gitignore.txt

```diff
@@ -10,3 +10,4 @@ tmp
 .project
 .settings
 Thumbs.db
+plugins/ti.alloy/hooks/alloy.js
```

Comment:
I think we'd want to ignore `plugins/ti.alloy` and not the specific file? There's a couple other files under there that can change (hooks/readme, hooks/deepclean.js, plugin.py)

Reply:
yep that actually makes more sense! Will update the PR